src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    private static final FigureSupplier FIGURE_SUPPLIER = new FigureSupplier();
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[randomInt()];
+        for (int i = 0; i < figures.length; i++) {
+            figures[i] = FIGURE_SUPPLIER.getRandomFigure();
+        }
+        for (int i = 0; i < figures.length; i++) {
+            System.out.println(figures[i].toString());
+        }
+    }
+
+    public static final int randomInt() {
```

Comment:
Why does this method `static` and `final`? What about method naming (you can read about method naming [here](https://mate-academy.github.io/style-guides/java/java.html#s5.2.3-method-names))

Reply:
@Override

public static final int randomInt() {
renamed 
private static int randomInt() {